src/components/App/App.js

```diff
@@ -1,11 +1,34 @@
-import React from 'react' 
+//Imports
+import React from 'react'
+import { Route } from 'react-router-dom'
+
+//Components Imports
+import Header from '../Header/Header'
 import Home from '../Home/Home'
+import Login from '../Login/Login'
+import UserLanding from '../User-Landing-Page/User-Landing'
+
+//Styling Imports
 import './_App.scss'
 
 const App = () => {
   return (
     <main>
-      <Home/>
+      <Header />
+      <Route
+      exact
+      path="/" component={Home}
+      />
+      <Route
+      exact
+      path="/login"
+      render={() => <Login/>}
+      />
+      <Route
+      exact
+      path='/user-landing'
+      render={() => <UserLanding/>}
```

Comment:
Going forward, will we need App useState hook to track the user id (to pass to landing page)? Possibly pass the hook through to login page?

Reply:
I don't think so, personally. I'd rather that ownership be limited to as many places as possible. Additionally, there might be some other hooks that could be beneficial. I'm thinking useContext.